Automattic/wp-calypso
# State: Add reducer and selector for WP core update status

This PR adds reducer and selector for WordPress core update status. We'll need this one after an update has completed (being successfully or not), because in `SiteIndicator` we want to show different messages upon success and error.

So, this PR hooks into the existing `sites/updates` reducer and adds a subtree to track the status. It also introduces a selector to fetch that status, storing it on a per-site basis.

To test:
* Checkout this branch.
* Verify all updates tests pass:
```
npm run test-client client/state/sites/updates/
```
* Verify the new selector tests pass:
```
npm run test-client client/state/selectors/test/get-site-core-update-status.js
```
